variables.tf

```diff
@@ -94,10 +85,34 @@ variable "environment" {
   default     = {}
 }
 
-variable "event" {
-  description = "Event source configuration which triggers the Lambda function. Supported events: cloudwatch-scheduled-event, dynamodb, s3, sns"
-  type        = map(string)
-  default     = {}
+variable "schedule_expression" {
+  description = "value"
```

Comment:
We'll want to provide a description of these new variables so folks know what they're for and in what context/architecture they need to be provided. 

Right now we have the ability to config an s3 bucket trigger with an empty string for an id and arn. The module we're using _should_ catch it and fail out but it's not documented anywhere if that's the only time we're using a variable like bucket_id. Terraform doesn't support variable validation to that level (yet) but we can document some of the dependencies of a variable in this file (and the readme once terraform-docs is configured) to help keep our support/documentation requests low.